browser/scripts/ui.js

```diff
@@ -67,6 +69,22 @@ VizorUI.prototype.showLoadingIndicator = function() {
 // VizorUI.prototype.openModal
 // VizorUI.prototype.closeModal
 
+VizorUI.prototype.setPageTitle = function() {
+	var graphname = E2.app.path;
+	var isLoggedIn = E2.models.user.get('username');
+	var newTitle = document.title;
```

Comment:
this doesn't update when the path changes, eg. by clicking `Duplicate` or auto-forking. call `setPageTitle()` also from `setupEditorChannel()` or similar